web/explorer/src/components/SettingsPanel.vue

```diff
@@ -20,9 +20,9 @@
                     />
                     <label for="showLibraryRules">
                         <span v-if="libraryRuleMatchesCount > 1">
-                            Show {{ libraryRuleMatchesCount }} library rule matches
+                            Show {{ libraryRuleMatchesCount }} distinct library rules
                         </span>
-                        <span v-else>Show 1 library rule match</span>
+                        <span v-else>Show 1 distinct library rule</span>
```

Comment:
is this also shown for 0 library rule matches?

Reply:
Good catch, I will add another `v-else-if count === 0`.

Reply:
Should now be resolved. Feel free to test with https://mandiant.github.io/capa/explorer/#/?rdoc=https://dpaste.org/Ox4tu/raw once merged.